data-plane/config/source/500-dispatcher.yaml

```diff
@@ -15,14 +15,16 @@
 # limitations under the License.
 
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
```

Comment:
Aren't there impacts with this? versus a normal deployment?

I guess the "v2" source was also Stateful set?

Reply:
Right - v2 source was a Stateful set - yes. The HA scheduler is dependent on this.